packages/react/objects/facets/Sticky.ts

```
@@ -1,11 +1,16 @@
-}
-
 export enum StickyPosition {
   TOP = 'TOP',
   BOTTOM = 'BOTTOM',
 }
 
+export const getStickyClassName = (stickyType?: string): string => {
+  switch (stickyType) {
+    case StickyPosition.BOTTOM:
+      return 'sticky-bottom'
+    case StickyPosition.TOP:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)